workshop-instructions/instructions.md

```diff
@@ -0,0 +1,106 @@
+**Table of Contents**
+
+- [Logging into the Oscar Virtual Environment](#logging-into-the-oscar-virtual-environment)
+- [Launching HNN-Core GUI via Oscar](#launching-hnn-core-gui-via-oscar)
+- [Launching HNN-Core (Python) via Oscar](#launching-hnn-core-python-via-oscar)
+- [Cloning the HNN-Data Repository](#cloning-the-hnn-data-repository)
+- [Installing HNN-Core GUI on Your Local Machine via Conda](#installing-hnn-core-gui-on-your-local-machine-via-conda)
+
+
+## Logging into the Oscar Virtual Environment
+1. Go to Oscar-on-Demand (OOD): [ood.ccv.brown.edu/pun/sys/dashboard/batch_connect/sessions](https://ood.ccv.brown.edu/pun/sys/dashboard/batch_connect/sessions)
+
+2. Select a username from the <a href="https://docs.google.com/spreadsheets/d/1NQuCULv6Nmo1n7cHnsD5ZnEYtnxPeYUWzRBvaXFvliA/edit?usp=sharing">following spreadsheet</a> and add an "X" to the appropriate cell. The "X" is to indicate that the username you selected is "in use" so another participant doesn't try to log in with the same username/bypass code
+
+3. Enter your selected username into 'Username' field of the login page
+
+4. Enter the password: HNNws2024*
+
+<div style="max-width: 600px;">
+
+![](imgs/oscar_login.png)
+
+</div>
+
+
+5. From the DUO authentication screen, select "Other options" and then choose "Bypass code"
+
+<span style="max-width:800px">
+
+| ![](imgs/duo_01.png) | ![](imgs/duo_02.png) | ![](imgs/duo_03.png) |
+|-----------------------------|-----------------------------|-----------------------------|
+
+6. Enter the bypass code associated with the username you selected in Step 2
+
+</span>
+
+
+## Launching HNN-Core GUI via Oscar
+With the Oscar desktop instance open, open a terminal and type the following to activate the environment.
```

Comment:
Needs directions on how to launch the desktop application. 

2. Choose the "**Desktop (Advanced)**" application. Note that this is different form the regular "Desktop" app.
3. You will see a form. Enter the following in the fields.
    1. Account: **[Leave Blank]**
    2. Desktop Environment: **xfce**
    3. Number of Hours: **3**
    4. Partition: **batch**
    5. Num Cores: **8**
    6. Num GPUs: **1** [This number does not matter]
    7. Memory (GB): 8
    8. Reservation: **hnn-workshop**
    9. Slurm Features: **[Leave Blank]**
4. Click "Launch" . It may take a minute before the “Launch Desktop” button appears.
2. Once Desktop is launched, open a Terminal via the “Terminal Emulator” at the bottom
of the Desktop or via the Applications drop-down at the top-left of the Desktop.

Reply:
These are the specs they recommended last time. I'm testing to see if the number of cores is appropriate.

Reply:
Awesome, thank you! I'd just been using the regular Desktop app 😅

Reply:
If you want to test using **Advanced**, just leave the `Reservation` field blank. The reservation just gives priority to more quickly launch a session during the reservation window. That reservation doesn't start until a few hours before the workshop though.  What time does the workshop start?

Reply:
It starts 8:30 AM Sydney time on Thursday morning, which should be 6:30 PM EST on Wednesday